### DISCUSSION TITLE
トラックボール動作せず

### DISCUSSION THREAD

**Discussion by na260** [Yowkees/keyball#65] [Q&A] [answered]
こちらで挙げられている内容
https://github.com/Yowkees/keyball/discussions/51#discussioncomment-1044412
1-5まで何度もためしてみたのですが、うまくいきません。
症状としてはこちらと同様、勝手に左下にカーソルが動いてしまい、トラックボールを動かしても反応しません。
申し訳ありませんがセンサーの交換していただくことは可能でしょうか？

**Comment by Yowkees** [comment, marked as answer]
了解しました。

こちらで半田付けして動作確認済のセンサー基板をお送りさせてください。
無事に新しい基板で正常に動作しましたら、不具合のあるセンサー基板を着払いでご返送いただければ幸いです。

ツイッターにてYowkeesをフォローしていただけますか？
ツイッターのDMにてご住所等やり取りをさせてください🙏

**Reply by na260** [reply]
ご考慮ありがとうございます。
twitter フォローさせていただきました。
hirokiICELANDというアカウントです。
DMが送れないようなのですが
他に何かしなければいけませんでしょうか？

**Comment by na260** [comment]
本日、到着し、無事動作することを確認しました。
ありがとうございます。
お手数をおかけしました。
